TTS/tts/models/xtts.py

```diff
@@ -396,7 +396,7 @@ def inference_with_config(self, text, config, ref_audio_path, language, **kwargs
         inference with config
         """
         assert (
-            language in self.config.languages
+            "zh-cn" if language == "zh" else language in self.config.languages
```

Comment:
It will return "zh-cn"  when language == "zh" and it will check if not. It is ok but for both conditional returns the same data type it should be  True if language == "zh" else language in self.config.languages.